### DISCUSSION TITLE
Multiple database (mysql) with same scheme

### DISCUSSION THREAD

**Discussion by rymca** [prisma/prisma#10256] [Q&A] [answered]
Hello,
I wanted to ask you the best way to implement this situation with Prisma.

I have multiple database with same scheme (one for each customer), 
when the customer logs in, all subsequent calls are managed on his database.

How i can manage this approach, 
am I forced to create multiple Prismaclients? 
all db have the same scheme, it would be enough to call the connection url ..

thank you all

**Comment by austincrim** [comment, marked as answer]
Hi @rymca 👋, thanks for the question.

> am I forced to create multiple Prismaclients?

This is correct. You can dynamically provide a connection string when instantiating a `PrismaClient`.

```ts
const client1 = new PrismaClient({
  datasources: {
    db: {
      url: 'connection string'
    }
  }
})

const client2 = new PrismaClient({
  datasources: {
    db: {
      url: 'a different connection string'
    }
  }
})
```

You might also want to read through [this issue](https://github.com/prisma/prisma/issues/2443) and add a +1 or more details about your use case.

Let me know if you have any further questions!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
